src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+  
+@font-face {
+  font-family: sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
```

Comment:
There should be more than one @font-face. You can find a hint in the checklist
![image](https://user-images.githubusercontent.com/99403198/235618465-adb27908-67ea-4197-abb1-686bdfb95b57.png)